src/System.Console/tests/Common/System/IO/InterceptStreamWriter.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
How about putting it in src/Common/tests instead of in src/System.Console/tests/Common?